hera_cal/abscal.py

```diff
@@ -3526,6 +3527,8 @@ def post_redcal_abscal_run(data_file, redcal_file, model_files, raw_auto_file=No
         refant: tuple of the form (0, 'Jnn') indicating the antenna defined to have 0 phase. If None, refant will be automatically chosen.
         clobber: if True, overwrites existing abscal calfits file at the output path
         add_to_history: string to add to history of output abscal file
+        write_delta_gains: write the degenerate gain component solved by abscal so a separate file specified by output_file_delta
```

Comment:
I'd specific that the output file has flags and quality arrays that match the output abscal file

Reply:
I've added this.